index.js

```diff
@@ -336,6 +336,22 @@
     });
   };
 
+  //  prop :: Accessible a => String -> a -> b
+  var prop =
+  def('prop',
+      {a: [Accessible]},
+      [$.String, a, b],
+      function(key, obj) {
+        var boxed = Object(obj);
+        if (key in boxed) {
+          return boxed[key];
+        } else {
+          throw new TypeError('‘prop’ expected object to have a property ' +
+                               'named ‘' + key + '’; ' + R.toString(obj) +
+                               ' does not');
```

Comment:
Sorry that I wasn't clear. By :arrow_left: I meant we should nudge the line one character to the left. We had:

```javascript
throw new TypeError('...' +
                     '...' +
                     '...');
```

I wonder whether we could have the linter highlight alignment issues.